09-writing-advanced-plans/README.md

```diff
@@ -100,7 +102,8 @@ Bolt supports a powerful extension mechanism via Puppet functions. These are fun
       $result = run_command('df', $nodes)
       return $result.map |$r| {
         $line_count = $r['stdout'].split("\n").length - 1
-        "${$r.target.name} has ${$line_count} volumes"
+        $node_name = $r.target.name
+        "$node_name has ${$line_count} volumes"
```

Comment:
why the intermediate variable assignment? it's only used once

Reply:
Easier to read for beginners . I actually meant to change the ${$line_count} to $line_count also.  The double $ will freak some people out.  I will push an update for the line_count.